src/app/tests/suites/certification/Test_TC_SC_4_1.yaml

```diff
@@ -400,9 +411,9 @@ tests:
                 value: nodeId
 
     - label:
-          "Step 3a: DUT put in Commissioning Mode using Open Basic Commissioning
-          Window command, starting advertising Commissionable Node Discovery
-          service using DNS-SD"
+          "Step 3a: If (CADMIN.S.C01.Rsp) present, DUT put in Commissioning Mode
```

Comment:
You can also add a PICS on step 3 so you don't need to reboot for no reason.

Also, all of step 2 should be gated by the PICS because you won't be advertising as commissionable if the device commissioning window isn't open. The fact that this is passing suggests that you may have a problem with your MCORE PICS.

Reply:
Step 3: Unfortunately we need the reboot at step 3 as step 4 is expecting a new randomly selected ID different to step 2 - `DNS-SD instance name must be 64-bit randomly selected ID expressed as a sixteen-char hex string with capital letters and must be different from the one at step 2` so a reboot will change the ID, I know not necessary but the test plan does not have a PICS removing step 4 if Basic Commissioning is not supported.

Step2: We are not advertising as commissionable hence gating steps 2g and 2n, maybe it would be better if I was checking these steps had a commissionable state = 0? and Step2 (from the Test Plan is not gated with CADMIN.S.C01.Rsp)

These current PR changes I believe are compatible with R1.3, R1.4 and beyond and negates the need for Test Plan changes.